R/tags.R

```diff
@@ -356,7 +366,8 @@ tag <- function(`_tag_name`, varArgs) {
   structure(
     list(name = `_tag_name`,
       attribs = attribs,
-      children = children),
+      children = children,
+      .noWS = .noWS),
```

Comment:
Try omitting the `.noWS` totally if NULL and see if that hashes in the test revert so we don't change everything.